.gitattributes

```diff
@@ -0,0 +1,12 @@
+/.atoum.php           export-ignore
+/.bootstrap.atoum.php export-ignore
+/.coke                export-ignore
+/.gitattributes       export-ignore
+/.gitignore           export-ignore
+/.travis.yml          export-ignore
+/bin                  export-ignore
+/Dockerfile           export-ignore
+/docker               export-ignore
+/docker-compose.yml   export-ignore
+/phpunit.xml.dist     export-ignore
```

Comment:
this file does not exist as the bundle seems to use atoum for testing instead of phpunit